src/parseSection.js

```diff
@@ -68,6 +72,18 @@ const splitSection = (section) => {
     }
 };
 
+/**
+ * Searches for enum values of an argument
+ * @param {string} option - argument with description from Options section
+ * @param {object} context - internal config
+ * @return {array} - enum values of argument
+ */
+const getEnum = (option, context) => {
+    const match = option.match(context.regexp.enumValues.enum);
+    return match ?
+        match[1].split(context.regexp.enumValues.split) : [];
```

Comment:
Please, check the `match` const for `null` or/and `empty`

Reply:
@svoboda-rabstvo, yes, it's not necessary in this function. It is necessary only on final function of parser, where template is filling. So, templatizer function should have (and has, just look in that PR) that check if enum empty despite of checking it in function `getEnum()`.

Reply:
Ok